code/modules/mining/drilling/scanner.dm

```diff
@@ -2,7 +2,7 @@
 	name = "ore detector"
 	desc = "A complex device used to locate ore deep underground."
 	icon = 'icons/obj/device.dmi'
-	icon_state = "forensic0-old" //GET A BETTER SPRITE.
+	icon_state = "mining1" //GET A BETTER SPRITE.
```

Comment:
This comment probably shouldn't exist. I'll remove it in a coming commit- awaiting input on the t-ray sprites first.